setup.py

```diff
@@ -10,6 +10,7 @@
     'python-dateutil',
     'prompt_toolkit>=2',
     'requests',
+    'typing_extensions'
```

Comment:
This should be

```
'typing_extensions' ; python_version<'3.8'
```

...to avoid pulling in the `typing_extensions` dependency on later Pythons.

Otherwise looks great, thanks! :)

Reply:
Good point. There are probably no typing changes from Python >=3.9 that need to be pulled in via `typing_extensions` foreseeable future from what I can tell.

Reply:
Actually, on second thoughts, I think `typing_extensions` should be pulled in regardless of Python version to avoid having to perform Python version checks when importing these annotations. 

`typing_extensions` already performs this check for us: https://github.com/python/typing_extensions/blob/fb77519b198f9d610ee8f674e27b8ce1da00f61f/src/typing_extensions.py#L228-L229

Furthermore, leaving it in for all Python versions makes it easier to write code that is compatible with all supported versions of Python, e.g:

```py
from typing_extensions import Literal, Protocol

class MyProto(Protocol):
    def foo() -> Literal["bar"]: ...
```

Instead of:

```py
import sys
if sys.version_.info < (3, 8):
    from typing_extensions import Literal, Protocol
else:
    from typing import Literal, Protocol

class MyProto(Protocol):
    def foo() -> Literal["bar"]: ...
```

Right now, the `.pyi` modules will show import errors in IDEs if pip installing `mreg_cli` on Python >=3.8, due to `typing_extensions` not being installed. This of course has no runtime implications\*, as these files are ignored by the interpreter, but it stands to reason that including such a small dependency for all versions should be a small concession to make to avoid having to write code like in the second example. 

\* Until `Literal`, `Protocol`, etc. are attempted to be imported into`.py` modules from `typing_extensions` at a later point.

Reply:
Oh, I thought the namespace was the same.  I agree that using `typing_extensions` unconditionally and consistently is easier.  LGTM!  :-)